digitalocean/resource_digitalocean_kubernetes_cluster.go

```diff
@@ -87,6 +87,27 @@ func resourceDigitalOceanKubernetesCluster() *schema.Resource {
 
 			"tags": tagsSchema(),
 
+			"maintenance_policy": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"day": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"duration": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"start_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
```

Comment:
```suggestion
			"maintenance_policy": {
				Type:     schema.TypeList,
				MinItems: 1,
				MaxItems: 1,
				Optional: true,
				Elem: &schema.Resource{
					Schema: map[string]*schema.Schema{
						"day": {
							Type:     schema.TypeString,
							Optional: true,
						},
						"start_time": {
							Type:     schema.TypeString,
							Optional: true,
						},
						"duration": {
							Type:     schema.TypeString,
							Computed: true,
						},
					},
```

Generally, you want to use `Computed: true` for attributes that the user can not configure. So it is correct to set that for `duration` since it is read only, we don't want to set that here for `day` or `start_time`.

https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#computed

It's probably a good idea to set `MinItems` and `MaxItems` as well.